src/olympia/lib/settings_base.py

```diff
@@ -1759,10 +1752,20 @@ def read_only_mode(env, retry_after=None):
     path('static'),
 )
 
-NETAPP_STORAGE = TMP_PATH
-GUARDED_ADDONS_PATH = os.path.join(ROOT, 'guarded-addons')
+# Path related settings. In dev/stage/prod `NETAPP_STORAGE_ROOT` environment
+# variable will be set and point to our NFS/EFS storage
+STORAGE_ROOT = env('NETAPP_STORAGE_ROOT', default=path('storage'))
+
+ADDONS_PATH = os.path.join(STORAGE_ROOT, 'files')
+GUARDED_ADDONS_PATH = os.path.join(STORAGE_ROOT, 'guarded-addons')
+GIT_FILE_STORAGE_PATH = os.path.join(STORAGE_ROOT, 'git-storage')
+
+SHARED_STORAGE = os.path.join(STORAGE_ROOT, 'shared_storage')
```

Comment:
I think the "shared" part is there for historical reasons, I thought I'd name it accordingly… happy to hear other ideas :)